content-learn/mastering-meshery-istio/performance-of-service-mesh/istio/circuit-breaking.mdx

```diff
@@ -0,0 +1,117 @@
+---
+docType: "Chapter"
+chapterTitle: "Circuit Breaking"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 4
+lectures: 12
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+
+<ChapterStyle>
+
+In this chapter we will configure circuit breaking using Istio. Circuit breaking allows developers to write applications that limit the impact of failures, latency spikes, and other undesirable effects of network peculiarities. This task will show how to configure circuit breaking for connections, requests, and outlier detection.
+
+<h2 class="chapter-sub-heading">Preparing for circuit breaking</h2>
+<br />
+
+Before we can configure circuit breaking, please try to access the `product page` app from within `Meshery` to ensure all the calls are making it through **without** errors as we did in [Observability chapter](/learn-ng/mastering-meshery-istio/performance-of-service-mesh/istio/observability) ([see screenshot](https://raw.githubusercontent.com/leecalcote/istio-service-mesh-workshop/feature/blend-in-meshery/lab-7/img/meshery_initial_load_test.png)).
+
+<h2 class="chapter-sub-heading"> Configure circuit breaking</h2>
+
+<br />
+Now that we have the needed services in place, it is time to configure circuit breaking
+using a destination rule.
+
+Using Meshery, navigate to the Istio management page:
+
+1. Enter `default` in the `Namespace` field.
+2. Click the (+) icon on the `Apply Custom Configuration` card and paste the configuration below.
+
+<small>Manual step for can be found [here](#appendix)</small>
```

Comment:
This doesn't work, change it to `h4` wrapped in `a` tag.

Reply:
Is it really necessary, i think it would be better without it, since the manual installation is shown at last

Reply:
This line contains the link to the manual installation. And, secondly if you check the preview you will notice that the `small` is not being parsed. So, we need to change it. If not `h4` we need to change to `p` at least and have the link around `here` which leads to the manual installation section below.

Reply:
Yes, It makes sense, i will change it 👍🏽